app/assignment/directives/instructorAssignmentTable.js

```diff
@@ -534,87 +534,112 @@ let instructorAssignmentTable = function ($rootScope, AssignmentActionCreators,
 								rowsSinceHeaderWasAdded++;
 							}
 						}); // Ending loop over courses
+					}
 
-						//Add 'The Staff'
+					element.append(coursesHtml);
 
-						let courseHtml = "";
-						courseHtml += "<div class=\"course-list-row\">";
-						courseHtml += "<div class=\"description-cell\">";
-						courseHtml += "<div>";
+					// Manually activate bootstrap tooltip triggers
+					$('body').tooltip({
+						selector: '[data-toggle="tooltip"]'
+					});
+				}
+			};
 
-						courseHtml += "<span style=\"margin-right:5px;\">";
+			scope.renderStaffTable = function () {
+				// Clear the table
+				$('.tooltip').remove();
+				element.empty();
 
-						// Instructor assignmentCompleted UI
-						courseHtml += "</span>";
-						courseHtml += "<div><strong>";
-						courseHtml += "The Staff";
-						courseHtml += "</strong>";
-						courseHtml += "</div>";
+				var instructorTypeHeader = '<div class="instructor-type-header">' + "Instructor TBA" + '</div>';
+				element.append(instructorTypeHeader);
 
-						// Instructor Comment UI
-						courseHtml += "<div class=\"description-cell__comment-btn-container hidden-print\">";
-						courseHtml += "</div>";
+				var header = scope.renderHeader();
+				element.append(header);
 
-						// If they don't have any teachingCallResponses, there won't be any unavailabilities to show
-						courseHtml += "<div class=\"description-cell__avail-btn-container\">";
-						courseHtml += "</div>";
-						courseHtml += "</div>";
-						courseHtml += "</div>"; // end description-cell
+				//Add 'The Staff'
+				var courseHtml = "";
 
-						// Loop over active terms
+				// Display message if table is empty
+				if (scope.view.state.courses.ids == 0) {
+					courseHtml += "<div class=\"course-list-row\">";
+					courseHtml += "<div class=\"course-description-cell empty-table-message\">";
+					courseHtml += "No courses have been added to the schedule";
+					courseHtml += "</div>";
+				}
+				else {
+					courseHtml += "<div class=\"course-list-row\">";
+					courseHtml += "<div class=\"description-cell\">";
+					courseHtml += "<div>";
+					courseHtml += "<span style=\"margin-right:5px;\">";
+
+					// Instructor assignmentCompleted UI
+					courseHtml += "</span>";
```

Comment:
Put this on line 573: `courseHtml += "<span style=\"margin-right:5px;\"></span>";`